msal/src/main/java/com/microsoft/identity/client/internal/MsalUtils.java

```diff
@@ -234,8 +237,26 @@ public static int getExpiryOrDefault(String expiresIn) {
      * @param url
      * @return
      */
+    @Deprecated
     public static boolean hasCustomTabRedirectActivity(@NonNull final Context context,
                                                        @NonNull final String url) {
+        try {
+            return validateCustomTabRedirectActivity(context, url);
+        } catch (final MsalClientException e) {
+            return false;
+        }
+    }
+
+    /**
+     * validateCustomTabRedirectActivity - Ensures that the developer has properly configured their
+     * AndroidManifest to expose the BrowserTabActivity.
+     *
+     * @param context
+     * @param url
+     * @return
+     */
+    public static boolean validateCustomTabRedirectActivity(@NonNull final Context context,
```

Comment:
Does this one need to be public?

Reply:
I can move it to the class and make it private but then we need to figure out something for this other method that I deprecated: https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/72ed05116f398f0d763544410d88d419b0040aa3/msal/src/main/java/com/microsoft/identity/client/internal/MsalUtils.java#L241

Can choose to just delete it and increment a major version. Thoughts?

Reply:
The deprecated one is fine as is.  We need a task to remove it in the next MV (in the fall).

Reply:
So basically I modified the deprecated API to call my new method (to avoid code duplication). So if I move my new method to some other class and make it private then I won't be able to access it here inside the deprecated method. One thing I can do is just duplicate the code in both the classes for now until the next MV.

Reply:
Leaving the code duplicated seems like the best path, otherwise we create another public API.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1357/commits/91fe570fc177b9cc298d2d85147cebff4f98344d